tidb-cloud/import-csv-files.md

```diff
@@ -103,60 +103,42 @@ To import the CSV files to TiDB Cloud, take the following steps:
-
-        > **Note:**
-        >
-        > When you use this feature, one import task can only import data to a single table at a time. If you want to use this feature to import data into different tables, you need to import several times, each time specifying a different target table.
-
-        To modify the file pattern, click **File Pattern**, specify a custom mapping rule between CSV files and a single target table in the following fields, and then click **Save**. After that, the data source files will be re-scanned using the provided custom mapping rule.
-
-        - **Source file name**: enter a pattern that matches the names of the CSV files to be imported. If you have one CSV file only, enter the file name here directly. Note that the names of the CSV files must include the suffix `.csv`.
-
-            For example:
-
-            - `my-data?.csv`: all CSV files starting with `my-data` and one character (such as `my-data1.csv` and `my-data2.csv`) will be imported into the same target table.
-            - `my-data*.csv`: all CSV files starting with `my-data` will be imported into the same target table.
-
-        - **Target table name**: enter the name of the target table in TiDB Cloud, which must be in the `${db_name}.${table_name}` format. For example, `mydb.mytable`. Note that this field only accepts one specific table name, so wildcards are not supported.
+    > **Note:**
+    >
+    > For the configurations of separator and delimiter, you can use both alphanumeric characters and certain special characters. The supported special characters include `\t`, `\b`, `\n`, `\r`, `\f`, and `\u0001`.
 
-    - **Table Filter**: If you want to filter which tables to be imported, you can specify table filter rules in this area.
+5. You can choose to **Import into Pre-created Tables**, or **Import Schema and Data from S3**.
 
-        For example:
+6. If you want to modify the file patterns, click **Advanced Settings** and then click **Mapping Settings**.
 
-        - `db01.*`: all tables in the `db01` database will be imported.
-        - `!db02.*`: except the tables in the `db02` database, all other tables will be imported. `!` is used to exclude tables that do not need to be imported.
-        - `*.*` : all tables will be imported.
+    > **Note:**
+    >
+    > When you use this feature, one import task can only import data to a single table at a time. If you want to use this feature to import data into different tables, you need to import several times, each time specifying a different target table.
 
-        For more information, see [table filter syntax](/table-filter.md#syntax).
+    To modify the file pattern, specify a custom mapping rule between CSV files and a single target table in the following fields, and then click **Confirm**. After that, the data source files will be re-scanned using the provided custom mapping rule.
 
-5. Click **Next**.
+    - **Target Database**: enter the name of the target database in TiDB Cloud.
```

Comment:
Not the `the target database in TiDB Cloud`, but the target databases user selected